grafana/resource_team.go

```diff
@@ -158,6 +158,11 @@ func ReadMembers(d *schema.ResourceData, meta interface{}) error {
 		if teamMember.Email == "admin@localhost" {
 			continue
 		}
+		// Labels store information about auth provider used to sync the team member.
+		// Team synced members should be managed through team_external_group resource and should be ignored here.
+		if len(teamMember.Labels) > 0 {
```

Comment:
I'd use a parameter for that. `ignore_externally_synced_users` or something. And default it to true. It's a way of documenting the behavior